main.cpp

```diff
@@ -3,10 +3,11 @@
  * Copyright © 2011,2012 Aleksey Cherepanov <aleksey.4erepanov@gmail.com>.  See LICENSE.
  */
 
-#include <QtCore/QCoreApplication>
-#include <QtWidgets/QApplication>
+#include <QCoreApplication>
+#include <QApplication>
+#include <QDir>
 #include "mainwindow.h"
-
+#include "translator.h"
```

Comment:
Let's move private file includes to the top and the Qt ones bellow. As per the coding style rules we should arrange include directives in descending order of specificity. http://doc.qt.digia.com/qtcreator-extending/coding-style.html#including-headers